### DISCUSSION TITLE
Tracking tags

### DISCUSSION THREAD

**Discussion by Sean4572435243** [dotnet/roslyn#63102] [Q&A]
Are there any tracking tags (e.g. a guid) compiled into the IL output that would allow for some form of deanonymizing of the developers or owners?

**Comment by svick** [comment]
When using deterministic mode, which is enabled by default when using the .Net Core/.Net 5+ SDK, [the set of inputs that affect the output of the compiler is fully specified](https://github.com/dotnet/roslyn/blob/main/docs/compilers/Deterministic%20Inputs.md).

I think that out of those, only paths can be used to identify the developer and even then only sometimes: e.g. if my code is in C:\Users\Svick\Documents\code, you can figure out I probably compiled the code, but if it's in C:\code, then you won't learn anything about me.

**Reply by Sean4572435243** [reply]
Ok great.  So you're not aware of any broader intentional security tracking/fingerprinting?  It just seems like this would be an obvious target for counterterrorism/counterespionage efforts.
